<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Meas</class>
 <widget class="QDialog" name="Dialog_Meas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLeft">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QCheckBox" name="EYE_WIDTH_chb">
         <property name="text">
          <string>EYE_WIDTH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="EYE_HEIGHT_chb">
         <property name="text">
          <string>EYE_HEIGHT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="EYE_HIGH_chb">
         <property name="text">
          <string>EYE_HIGH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="EYE_LOW_chb">
         <property name="text">
          <string>EYE_LOW</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="EYE_AMPLITUDE_chb">
         <property name="text">
          <string>EYE_AMPLITUDE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="EYE_PCROSS_chb">
         <property name="text">
          <string>EYE_PCROSS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="EYE_NCROSS_chb">
         <property name="text">
          <string>EYE_NCROSS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="EYE_EXTICTRATIO_chb">
         <property name="text">
          <string>EYE_EXTICTRATIO</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalRight" stretch="0,0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QCheckBox" name="EYE_QFACTOR_chb">
         <property name="text">
          <string>EYE_QFACTOR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="EYE_AOPTPWRDBM_chb">
         <property name="text">
          <string>EYE_AOPTPWRDBM</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_Meas</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_Meas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
